modules/shared/src/main/scala/io/constellationnetwork/schema/currencyMessage.scala

```diff
@@ -29,6 +30,8 @@ object currencyMessage {
 
     case object Owner extends MessageType("Owner")
     case object Staking extends MessageType("Staking")
+
+    implicit object OrderingInstance extends OrderBasedOrdering[MessageType]
```

Comment:
I'm not sure but there is already an ordering instance specified in `@derive(.., ordering)`, so you probably should remove the derivation if you want to use an explicitly defined one

Reply:
@gclaramunt in your tests did you find any difference between using the derive ordering macro vs. the scala builtin ordering implicit? We seem to have a mixed usage of either derive order, ordering, or derive order + implicit object declared order, but this is right we shouldn't be doing both